### ISSUE TITLE
Invalid argument: timeZone. Should be of type: String

### ISSUE THREAD

**Issue by akirabaruah** [Will-Renius/GoogleDocsJournal#3]
I see this error when selecting "Add-Ons > Dated Journal Entry > NewEntry" in in a new Google doc.

```
Invalid argument: timeZone. Should be of type: String
```

**Comment by Will-Renius** [comment]
What is timezone set to under Add-Ons > Dated Journal Entry >Change Date Format > Advanced Settings > Time Zone?

**Comment by akirabaruah** [comment]
I just re-installed the add-on. The Time Zone field now says "America/New_York" and the add-on seems to work as intended. 😅

If I remove the text in Time Zone and hit "save" I can reproduce the issue. Not sure if/how I got into that state when I installed it the first time, since I didn't touch the settings. Is it possible to add some logic to set a default timezone (GMT?) or prevent users from setting Time Zone to null?

I just realized that the code in this repo doesn't seem to match the extension for which I filed the bug. https://github.com/Will-Renius/DatedJournalEntry seems like the right one - if so, you might want to update the "Help > Report an issue" dialog to point to that repo's Issues instead of here.

My original issue seems to be resolved, but let me know if you'd like me to file other Issues for the points I mentioned above. Thanks!
